chatterbot/adapters/logic/closest_meaning.py

```diff
@@ -7,6 +7,10 @@
 
 
 class ClosestMeaningAdapter(BaseMatchAdapter):
+    """
+    The ClosestMeaningAdapter extends the BaseMatchAdapter to
+    find the response with the closest meaning.
```

Comment:
Pretty much any programmer will read `ClosestMeaningAdapter(BaseMatchAdapter)` as "The ClosestMeaningAdapter extends the BaseMatchAdapter" and based on it's name they will probably assume that it "find the response with the closest meaning".

Maybe we can provide some information that will help someone understand how the logic in this class generally works. In this case I might describe how this class functions as:

> This adapter selects a response by comparing the tokenized form of the input statement's text, with the tokenized form of possible matching statements. For each possible match, the sum of the Cartesian product of the path similarity of each statement is compared. This process simulates an evaluation of the closeness of synonyms. The known statement with the greatest path similarity is then returned.